lib/main.dart

```diff
@@ -11,38 +15,55 @@ class Home extends StatelessWidget {
       appBar: AppBar(
         backgroundColor: Color(0xFFEFEFEF),
         centerTitle: true,
-        flexibleSpace: Container(
-            decoration: BoxDecoration(
-                image: DecorationImage(
-          image: AssetImage("assets/ikhlas.png"),
-        ))),
+        title: Image.asset("assets/ikhlas.png"),
       ),
       body: Container(
         constraints: BoxConstraints.expand(width: 420, height: 162),
         decoration: BoxDecoration(color: Colors.teal[300]),
-        padding: new EdgeInsets.all(10.0),
-        child: Row(
-          mainAxisAlignment: MainAxisAlignment.spaceAround,
+        padding: new EdgeInsets.all(16.0),
+        child: Column(
           crossAxisAlignment: CrossAxisAlignment.start,
-          children: <Widget>[
-            Column(
+          children: [
+            Row(
+              mainAxisAlignment: MainAxisAlignment.spaceBetween,
               crossAxisAlignment: CrossAxisAlignment.start,
               children: <Widget>[
-                Text(
-                  "Next , Zohor",
-                  style: TextStyle(color: Colors.white, fontSize: 14),
-                ),
-                Text(
-                  '1:04 PM',
-                  style: TextStyle(color: Colors.white, fontSize: 32),
-                ),
-                Text(
-                  'Starts in 02:29:46',
-                  style: TextStyle(color: Colors.white, fontSize: 14),
+                Column(
+                  crossAxisAlignment: CrossAxisAlignment.start,
+                  children: <Widget>[
+                    Text(
```

Comment:
I tot we decided during our last discussion to create method and then call that method on the TextStyle?